src/torchcps/gnn.py

```diff
@@ -156,6 +156,7 @@ def __init__(
         n_taps: int,
         n_layers: int = 2,
         n_channels: int = 32,
+        architecture: str = "gcn",
```

Comment:
Why is this being changed?

Reply:
I think that's an artifact of when I was originally going to implement the GIN architecture within the GCN class before I realized that it would be better to just implement the GIN as a separate class, so I think it's safe to delete.